module-4/backend/week_one.md

```diff
@@ -5,18 +5,36 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+Basic syntax of javascript, and how logic looks basically the same in ruby and js. I also had a personal assignment using AJAX, similar to the module 3 lesson we had on AJAX. This helped me solidify some of the js/jQuery basics as well.
 2. What are some tools to help debug JavaScript code?
+Debugger, Console.log, and the many tools that the google chrome developer tools employ.
 3. What are some tools you need in order to unit test your JavaScript?
+Mocha is what we used in the prework and the week one assignments, but I know there are more options. Unit testing in the professional JS is prevalent where feature and model are not as much.
 4. What is the syntax for invoking a function?
+Parenthesis (empty or containing arguments) after a function name are required to invoke it, otherwise just the function itself is returned.
 5. What is CORS?
```

Comment:
Questions 5,6 & 10 are from an older version of this repo. Be sure to pull the latest for next week :)